### PR TITLE
Tweak metrics bundle names

### PR DESCRIPTION
**Pull request by tevans78**
Bundle names did not properly follow the OL naming conventions
#build

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

File: dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/internal/SmallryeMetricsCDIMetadata.java [hunk 1]
```diff
@@ -182,32 +198,22 @@ protected void unsetSharedLib(Library ref) {
...
+    }
+
+    private File resolveLibJar(String jarName, String jarVersionRange) throws FileNotFoundException {
         ContentBasedLocalBundleRepository cblb = BundleRepositoryRegistry.getInstallBundleRepository();
-        VersionRange vr = VersionRange.valueOf(VersionRange.LEFT_CLOSED + "1,5.0.100" + VersionRange.RIGHT_CLOSED);
-        /*
-         * Sometimes.. the first call returns a null file. Debug shows that the class
-         * loader was unable to load the ContentBasedLocalBundleRepository
-         */
-        cblb.selectBundle("lib/", "io.openliberty.micrometer", vr);
```
Comment (Channyboy) [inline dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/internal/SmallryeMetricsCDIMetadata.java:207]: Hm, sometimes the "first call" returns me a null file. That's why I had the duplicate call there. But I suppose if it passed the build then the build didn't encounter it.

**Review by Channyboy — APPROVED** [review]
lgtm

**Comment by Channyboy** [comment]
#libby

File: dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/internal/SmallryeMetricsCDIMetadata.java (hunk 1 above)
Comment (tevans78) [reply dev/io.openliberty.microprofile.metrics.5.0.internal/src/io/openliberty/microprofile/metrics50/internal/SmallryeMetricsCDIMetadata.java:207]: The PB did not encounter these issues. If it occurs again we should investigate to find another fix.

**Pull request merged**